src/acat/azext_acat/custom.py

```diff
@@ -0,0 +1,537 @@
+    class ListAcatReportWebhookWithDupAadToken(_AcatListReportWebhook.WebhookList):
+        CLIENT_TYPE = "AcatMgmtClient"
+
+    def _execute_operations(self):
+        self.pre_operations()
+        self.ListAcatReportWebhookWithDupAadToken(ctx=self.ctx)()
+        self.post_operations()
+
+
+class ShowAcatReportWebhook(_AcatShowReportWebhook):
+    class ShowAcatReportWebhookWithDupAadToken(_AcatShowReportWebhook.WebhookGet):
+        CLIENT_TYPE = "AcatMgmtClient"
+
+    def _execute_operations(self):
+        self.pre_operations()
+        self.ShowAcatReportWebhookWithDupAadToken(ctx=self.ctx)()
+        self.post_operations()
+
+
+class CreateAcatReportWebhook(_AcatCreateReportWebhook):
+    class CreateAcatReportWebhookWithDupAadToken(
+        _AcatCreateReportWebhook.WebhookCreateOrUpdate
+    ):
+        CLIENT_TYPE = "AcatMgmtClient"
+
+    def _execute_operations(self):
+        self.pre_operations()
+        self.CreateAcatReportWebhookWithDupAadToken(ctx=self.ctx)()
+        self.post_operations()
+
+    @classmethod
+    def _build_arguments_schema(cls, *args, **kwargs):
+        args_schema = super()._build_arguments_schema(*args, **kwargs)
+        from azure.cli.core.aaz import AAZListArg, AAZStrArg, AAZStrArgFormat
+
+        args_schema.content_type._required = False
+        args_schema.content_type._registered = False
+        args_schema.events._required = False
+        args_schema.events._registered = False
+        args_schema.payload_url._required = False
+        args_schema.payload_url._registered = False
+        args_schema.send_all_events._required = False
+        args_schema.send_all_events._registered = False
+        args_schema.status._required = False
+        args_schema.status._registered = False
+        args_schema.update_webhook_key._required = False
+        args_schema.update_webhook_key._registered = False
+        args_schema.webhook_key._required = False
+        args_schema.webhook_key._registered = False
+
+        args_schema.content_type_with_default = AAZStrArg(
+            options=["--content-type"],
+            arg_group="Properties",
+            help="content type",
+            required=False,
+            enum={"application/json": "application/json"},
+            default="application/json",
+        )
+        args_schema.events_with_default = AAZListArg(
+            options=["--events"],
+            arg_group="Properties",
+            help="under which event notification should be sent.",
+            required=False,
+            default=[],
+        )
+        args_schema.events_with_default.Element = AAZStrArg()
+        args_schema.payload_url_required = AAZStrArg(
+            options=["--payload-url"],
+            arg_group="Properties",
+            help="webhook payload url",
+            required=True,
+            fmt=AAZStrArgFormat(
+                pattern=r"^(http(s)?://)[\S]{0,64994}$",
+            ),
+        )
```

Comment:
You can set the _required property of argument directly
```suggestion
        args_schema.payload_url._required = True
```